### PR TITLE
Updating project setup to make steps more clear

### PR DESCRIPTION
**Pull request by 2020009-CPE**
_Description of what this PR is changing or adding, and why:_

_Issues fixed by this PR (if any):_ Fixes #ISSUE-NUMBER

## Presubmit checklist
- [ ] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [ ] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [ ] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.
- [ ] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [ ] This PR doesn’t contain automatically generated corrections (Grammarly or similar).

### CODE REVIEW

File: src/development/add-to-app/ios/project-setup.md [hunk 1]
```diff
@@ -3,36 +3,59 @@ title: Integrate a Flutter module into your iOS project
...
+  This is similar to the second option, but it provides
+  an alternative to distributing the large Flutter.xcframework.
+ 
+For examples using an app built with UIKit, 
+see the iOS directories in the [add_to_app code samples][]. 
+For an example using SwiftUI, see the iOS directory in [News Feed App][].
+ 
 ## System requirements
-
+
```
Comment (domesticmouse) [inline src/development/add-to-app/ios/project-setup.md:34]: Please trim trailing white space here and on following lines

**Review by domesticmouse — CHANGES_REQUESTED** [review]

**Comment by sfshaza2** [comment]
@jmagman, this PR needs wordsmithing and re-formatting but, before I dive into that, can you verify the changes?

**Comment by sfshaza2** [comment]
Also, @2020009-CPE, I can't land this without you signing the CLA.

**Review by jmagman — CHANGES_REQUESTED** [review]
@2020009-CPE Please fill out the description template as requested, particularly "Description of what this PR is changing or adding, and why:".  Can you give us a summary of what issues you hit and how the current documentation is unclear?  

Please also revert unnecessary whitespace changes as @domesticmouse suggested to make it easier to see what you actually changed.

**Comment by parlough** [comment]
It looks like these changes landed as part of https://github.com/flutter/website/pull/7828 by the original author of the attached commit.

Thanks for opening the PR though and let us know if you'd like to see any further changes or updates to the page, thanks!

**Pull request closed without merging**